styles.css

```diff
@@ -1,6 +1,6 @@
 @import "tailwindcss";
 
-@theme {
+@theme static {
```

Comment:
This ensures our default theme color variables are not tree-shaken. Adds a tiny bit of extra CSS to the final output, but lets you play with all the variables in the browser as needed.